server/error_recorder.go

```diff
@@ -0,0 +1,87 @@
+package server
+
+import (
+	"net/http"
+	"runtime/debug"
+	"time"
+
+	"github.com/bytebase/bytebase/api"
+	"github.com/labstack/echo/v4"
+)
+
+func errorRecorderMiddleware(err error, s *Server, c echo.Context, e *echo.Echo) {
+	req := c.Request()
+	res := c.Response()
+	var errorMessage string
+
+	// -----------------------------------Echo's DefaultHTTPErrorHandler BEGIN----------------------------------------
```

Comment:
The current comment is missing some context.

Worth commenting explicitly we copy from Echo's DefaultHTTPErrorHandler and explain why we do this

Reply:
Now we call `e.DefaultHTTPErrorHandler` inside `errorRecorderMiddleware` to reduce the code.